### PR TITLE
Mute other users

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
This adds a feature to allow readers to mute other user so their posts no longer show for them on that particular device.

## Why?
This was one of the requests made during the meeting with the moderators

## Link to supporting Trello card
https://trello.com/c/pELxIx7Z/1279-wishlist-mute-a-user

### CODE REVIEW

File: src/components/Comment/Comment.tsx [hunk 1]
```diff
@@ -384,12 +420,30 @@ export const Comment = ({
                       </button>
                     )}
                 </div>
-                <div>
+                <Row>
+                  {/* You can't mute unless logged in and you can't yourself */}
+                  {user && comment.userProfile.userId !== user.userId ? (
+                    <div className={buttonHeightOverrides}>
```
Comment (liywjl) [inline src/components/Comment/Comment.tsx:426]: the variations between this and `AbuseReportForm` cause the exact position to be slightly off, and for the on highlight to be different.

Screenshot 2020-04-02 at 13 09 23
Screenshot 2020-04-02 at 13 09 28

This should be fixed in another PR by changing `AbuseReportForm` to apply src foundations as it is due to src-foundations

Comment (oliverlloyd) [reply src/components/Comment/Comment.tsx:426]: Good spot. Agreed, lets move the abuse form over to src as well, and maybe do the same for Reply, Pick and Share at the same time?

Comment (liywjl) [reply src/components/Comment/Comment.tsx:426]: yes agreed!

**Comment by oliverlloyd** [comment]
> `writeMutes` is not used

Fixed 👍
